dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -384,6 +438,8 @@ def _get_dataset(value):
             self._input_datasets = tuple(input_datasets_list)
             self._input_names = tuple(input_names_list)
             self._input_layouts = tuple(input_layouts_list)
+            # Map it to integers
+            self._input_batched = tuple(1 if b else 0 for b in input_batched_list)
```

Comment:
Maybe this shows your intention more clearly:
```suggestion
            self._input_batched = tuple(int(b) for b in input_batched_list)
```

Reply:
done